frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -647,9 +648,16 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedRewardBase: Get<BalanceOf<Self>>;
 
-		/// Base deposit for a signed solution.
+		/// Fixed base deposit for a signed solution.
 		#[pallet::constant]
-		type SignedDepositBase: Get<BalanceOf<Self>>;
+		type SignedFixedDeposit: Get<BalanceOf<Self>>;
+
+		/// Increase factor of the geometric series for the base deposit computation.
+		///
+		/// If 0, then signed base deposit remains constant regardless of the submissions queue size
+		/// (and equal to `SignedFixedDeposit`).
+		#[pallet::constant]
+		type SignedDepositIncreaseFactor: Get<Percent>;
```

Comment:
This associated type is not useful if `SignedDepositBase` is not implementing a geometric distribution. Same can be said for the type above. These should be moved to `GeometricDepositBase`: 

```rust
struct GeometricDepositBase<Balance, Base: Get<Balance>, Inc: Get<Percent>>(...)
```

Reply:
Great suggestion, updated the PR.